cmd/lifecycle/restorer.go

```diff
@@ -90,10 +79,52 @@ func (r *restoreCmd) Privileges() error {
 }
 
 func (r *restoreCmd) Exec() error {
-	if r.supportsBuildImageExtension() {
-		if err := r.pullBuilderImageIfNeeded(); err != nil {
-			return cmd.FailErr(err, "read builder image")
+	var analyzedMD platform.AnalyzedMetadata
+	if _, err := toml.DecodeFile(r.AnalyzedPath, &analyzedMD); err == nil {
```

Comment:
here can you use the `platform.ReadAnalyzed` func?

Reply:
Updated in 5adcca54